crates/core/src/lib.rs

```diff
@@ -119,6 +119,8 @@ pub(crate) mod repository;
 /// Virtual File System support - allows to act on the repository like on a file system
 pub mod vfs;
 
+pub use log::Level as LogLevel;
+
```

Comment:
Is that an intended re-export? I assume we might want to use `tracing` in the future (especially when refactoring to `actors` or refactoring the threaded stuff), not sure if we want to have this as part of our API? 🤔

Reply:
is now changed by an own enum.